app/views/posts/index.html.erb

```diff
@@ -1,5 +1,7 @@
 <% @posts.reverse_each do |post| %>
-  <p><%= simple_format(post.message).html_safe %></p>
+  <p><%= simple_format(post.message).html_safe %>
+      <%= 'Posted ' + time_ago_in_words(post.created_at) + ' ago' %></p>
```

Comment:
Could you wrap the message and the time in separate <p> tags and wrap the whole thing in <div> tags? It will make no visual difference now but will help with styling later.

Reply:
Yeah course, good idea!